package.json

```diff
@@ -38,6 +38,7 @@
     "build:typings": "rm -rf ./typings && ./scripts/build-typings.sh && tsc --emitDeclarationOnly"
   },
   "dependencies": {
+    "@react-native-community/netinfo": "^5.6.2",
```

Comment:
Can you please move the dependency to `devDependencies` and `peerDependencies`. 
As you can see we avoid installing native dependencies so users can decide on the version that fit them.

Reply:
Oh ! Done :+1: